### DISCUSSION TITLE
pyMOR Community Meeting 2023-06-14

### DISCUSSION THREAD

**Discussion by HenKlei** [pymor/pymor#2068] [Announcements]
The next pyMOR Community Meeting will take place Wednesday, June 14, at 17:00 CEST and 11:00 EDT, online at:

link: https://bbb.mpi-magdeburg.mpg.de/meeting/pymor
password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [ ] NumFocus feedback? (@drittelhacker)
- [ ] pyMOR school (@pymor/all)
- [ ] Zenodo (@drittelhacker)
- [ ] AOB (@sdrave)

Moderator: @HenKlei

**Comment by lbalicki** [comment]
Minutes:
- NumFocus feedback
  - link is still broken on their website
  - Need to finish getting started PR from our side
- pyMOR school
  - still need to figure out financial aspects before finalizing registration form
- Zenodo
  - We have releases on there
  - would be good to add tags for DFG funded periods on there
- AOB
  - possibility for 1 year post-doc position for MOR with pyMOR
  - decission will be made after 5 minute pitch at an event
- Release
  - Would be good to finish a few of the bigger PRs for upcoming release
  - Push hard-freeze back by 1 week
 
Next Meeting: Wed, July 12th, 14:00 CEST, 8:00 EDT
Moderator: Petar
